tutorial_filtering.md

```diff
@@ -70,12 +70,12 @@ Just find the comment:
 and replace the filtering expression for something like this:
```

Comment:
Change required: 
for --> with:

and replace the filtering expression with something like this: